web/impact/impact/v1/views/cancel_office_hour_session_view.py

```diff
@@ -64,22 +63,31 @@ def get_response(success, detail):
     })
 
 
+def get_cancelled_by(mentor_name=None, staff=False):
+    to_finalist, to_mentor = mentor_name, 'You have'
```

Comment:
I don't love this shorthand style of writing assignments, but won't block on this.